frontend-ssr/components/exercise-submissions/Submission.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '../../styles/variables';
+@import '../../styles/utils.scss';
+.card {
+    margin: 1.2em 0;
+    padding: 0.8em;
+    transform: scale(1);
+    box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.1);
+    &:hover {
+        transform: scale(1.02);
+        transition: transform 0.3s cubic-bezier(0.075, 0.82, 0.165, 1);
+    }
+    .card_info {
+        width: 15em;
+    }
+    .card_avatar {
+        border-radius: 50%;
+        vertical-align: middle;
+    }
+    .card_el {
+        padding: 0.4em 0;
+        border: none;
+    }
+    .card_chapter {
+        min-width: 6em;
+    }
+    .card_date {
+        letter-spacing: 0.01em;
+    }
+    .card_status_pending,
+    .card_status_assigned {
+        min-width: 10em;
+        @extend %gradientBackground,
+        %greenGradientBackground;
+    }
+    .card_status_pending {
```

Comment:
2 separate rules for the same class name: `card_status_pending`?

Also, all class names should be `kebab-case`